elixir-ts-mode.el

```diff
@@ -613,6 +613,12 @@ Return nil if NODE is not a defun node or doesn't have a name."
     (save-excursion
       (newline 1 t))))
 
+(defcustom elixir-ts-mode-hook '()
```

Comment:
The convention is to specify 'nil' here.  Even though its the same it is still good to stick with conventions.